src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncMethodBuilderOverrideTests.cs

```diff
@@ -115,18 +115,18 @@ class C
 ";
             var compilation = CreateCompilationWithMscorlib45(source, parseOptions: TestOptions.Regular9);
             compilation.VerifyDiagnostics(
-                // (11,25): error CS8652: The feature 'async method builder override' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                // (11,25): error CS8773: Feature 'async method builder override' is not available in C# 9.0. Please use language version 10.0 or greater.
```

Comment:
It feels like even with the new scheme of "use LanguageVersionFacts.CSharpNext" we still have this churn of updating diagnostic IDs when we snap off a new language version.

When we snap a language version, should we also introduce an error code specific to 'C# vNext' which becomes the `ERR_FeatureNotAvailableInVersionN` error code when the language version gets snapped? (although this perhaps doesn't work for features like static abstracts in interfaces where we don't yet know what language version they are likely to ship in.)

Reply:
You're right that such diagnostics will still churn. We can explore some mitigations when we start working on C# 11.